### PR TITLE
broadcast error messages

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/Hooks/React-Query/useInfiniteCachedQuery.ts [hunk 1]
```diff
@@ -23,7 +23,9 @@ export function useInfiniteCachedQuery<
 ): UseQueryResult<TData, TError> {
     return useQuery(queryKey, queryFn, {
         ...options,
-        staleTime: CacheTime.TenHours,
+        staleTime: CacheTime.FiveMinutes,
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Hooks/React-Query/useInfiniteCachedQuery.ts:26]: 30min

File: src/apps/ScopeChangeRequest/Hooks/React-Query/useScopeChangeQuery.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { QueryFunction, QueryKey, useQuery, UseQueryOptions, UseQueryResult } from 'react-query';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Hooks/React-Query/useScopeChangeQuery.ts:1]: deprecate

File: src/apps/ScopeChangeRequest/Hooks/React-Query/useScopechangeMutation.ts [hunk 3]
```diff
@@ -7,7 +7,6 @@ import {
     MutationKey,
 } from 'react-query';
 import { useDataContext } from '../../../../Core/WorkSpace/src/Context/DataProvider';
-import { scopeChangeQueryKeys } from '../../Keys/scopeChangeQueryKeys';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Hooks/React-Query/useScopechangeMutation.ts:10]: deprecate

File: src/apps/ScopeChangeRequest/Api/PCS/getCommPkgsByIds.ts [hunk 4]
```diff
@@ -4,11 +4,15 @@ import { CommissioningPackage } from '../../Types/ProCoSys/CommissioningPackage'
...
 
-    return await procosys
-        .fetch(
-            `/api/CommPkg/ByCommPkgNos?plantId=PCS%24JOHAN_CASTBERG${commPkgNumbers.map(
-                (x) => `&commPkgNos=${x}`
-            )}&api-version=4.1`
-        )
-        .then((x) => x.json());
+    const res = await procosys.fetch(
+        `/api/CommPkg/ByCommPkgNos?plantId=PCS%24JOHAN_CASTBERG${commPkgNumbers.map(
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Api/PCS/getCommPkgsByIds.ts:8]: hennt ut PCS%24JOHAN_CASTBERG med useFacility() muilg plant id bør være innparameter

File: src/apps/ScopeChangeRequest/Api/PCS/getDisciplines.ts [hunk 5]
```diff
@@ -8,9 +8,13 @@ import { Discipline } from '../../Types/ProCoSys/discipline';
 export async function getDisciplines(): Promise<Discipline[]> {
     const { procosys } = httpClient();
 
-    const res: Promise<Discipline[]> = await procosys
-        .fetch('api/Library/Disciplines?plantId=PCS%24JOHAN_CASTBERG&api-version=4.1')
-        .then((x) => x.json());
+    const res = await procosys.fetch(
+        'api/Library/Disciplines?plantId=PCS%24JOHAN_CASTBERG&api-version=4.1'
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Api/PCS/getDisciplines.ts:12]: damme som over

File: src/apps/ScopeChangeRequest/Api/PCS/getDisciplineByCode.ts [hunk 6]
```diff
@@ -4,11 +4,12 @@ import { Discipline } from '../../Types/ProCoSys/discipline';
 export async function getDisciplineByCode(discCode: string): Promise<Discipline> {
     const { procosys } = httpClient();
 
-    const responseData: Discipline = await procosys
-        .fetch(
-            `api/Library/Discipline?plantId=PCS%24JOHAN_CASTBERG&code=${discCode}&api-version=4.1`
-        )
-        .then((x) => x.json());
+    const res = await procosys.fetch(
+        `api/Library/Discipline?plantId=PCS%24JOHAN_CASTBERG&code=${discCode}&api-version=4.1`
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Api/PCS/getDisciplineByCode.ts:8]: kan også bruke getClientContext() for så og hente ut facility der som inneholder procosys plantId

File: src/apps/ScopeChangeRequest/Api/STID/getDocumentsByTag.ts [hunk 7]
```diff
@@ -18,5 +18,12 @@ export async function getDocumentsByTag(
     const uri = `${instCode}/tag/document-refs`;
     const queryParameters = `tagNo=${encodeURIComponent(tagNo)}&noContentAs200=true`;
     const url = `${uri}?${queryParameters}`;
-    return await stidClient.fetch(url).then((response) => response.json());
```
Comment (Noggling) [inline src/apps/ScopeChangeRequest/Api/STID/getDocumentsByTag.ts:21]: husk å hente instcode også fra ClientContext

**Review by Noggling — APPROVED** [review]
Ellers ser alt ok ut ;)

**Pull request merged**